circle.yml

```diff
@@ -22,5 +22,24 @@ deployment:
     branch: master
     commands:
       - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker tag cyclus/cyclus:latest cyclus/cyclus:stable
-      - docker push cyclus/cyclus:stable
+      # Update cyclus/cylus:stable docker container
+      - docker tag cyclus/cyclus:latest cyclus/cyclus:stable # creation
+      - docker push cyclus/cyclus:stable # push to docker depot
+        ## DEB GENERATION
+          #ubuntu 14.04
+      - rm Dockerfile && ln -s docker/release-14.04/Dockerfile # place the correct Dockerfile
```

Comment:
Since these two dockerfiles are nearly identical....
Instead of linking to different copies, what about using sed to generate each one from a template by substituting in either "14" or "16"?